app/controllers/api/posts_controller.rb

```diff
@@ -19,10 +19,12 @@ def update
   private
 
   def create_params
-    params.require(:post).permit(:caption, :lat, :long, :file_path, :url, :skater_id, :up_vote_count, :down_vote_count)
+    params.require(:post).permit(:caption, :lat, :long, :file_path, :url, :spot_id, :up_vote_count, :down_vote_count).tap do |create_params|
+      create_params[:skater_id] = current_skater.id
+    end
   end
 
   def update_params
-    params.require(:post).permit(:caption, :lat, :long, :file_path, :url, :skater_id, :up_vote_count, :down_vote_count)
+    params.require(:post).permit(:caption, :lat, :long, :file_path, :url, :spot_id, :up_vote_count, :down_vote_count)
```

Comment:
remove spot_id, lat, long and url from here. 

Also, lets remove up_vote_count and down_vote_count from create and updat params methods.